lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
One more reason to move away from the KWARC branch and use core LaTeXML sooner rather than later.